drivers/arm/ws2812.c

```diff
@@ -84,17 +84,12 @@ void ws2812_setleds(LED_TYPE *ledarray, uint16_t leds) {
         sendByte(ledarray[i].g);
         sendByte(ledarray[i].r);
         sendByte(ledarray[i].b);
-    }
+#ifdef RGBW
+        sendByte(ledarray[i].w);
+#endif
+}
```

Comment:
```suggestion
    }
```

Reply:
That was already there .... :D

That said, ran `qmk cformat` on the files I touched.  Should be cleaner, now.